test/testdata/simpletests/datavolume.json

```diff
@@ -0,0 +1,8 @@
+{
+  "TaskDefinition": "datavolume",
```

Comment:
When I first wrote these (see earlier commits, [e.g.](https://github.com/euank/amazon-ecs-agent/commit/8c5340aae73e085e60899821fe046561b697900d#diff-96a6a497ed513990255825f39eb39558R7)), I had tags on tests which were meant to indicate that somewhat. This one would have had a 'volumes' tag. I removed the tags because they didn't have any functionality, but I still personally like the idea.

Would tags assuage this concern, or do you still think a short human readable description is the way to go?